lib/decidim/votings/feature.rb

```diff
@@ -15,11 +15,13 @@
   end
 
   # These actions permissions can be configured in the admin panel
-  feature.actions = %w(vote register_in_census)
+  feature.actions = %w(vote)
 
   feature.settings(:global) do |settings|
     # Add your global settings
     # Available types: :integer, :boolean
+    settings.attribute :remote_authorization, type: :boolean, default: false
```

Comment:
I'm not sure, but I think we can remove this setting. Just with a `nil` default value for `:remote_authorization_url` would allow to disable this feature.

Reply:
ok, it was only set to have a more understandable settings form. It is not needed.